### PR TITLE
Check allow-net in fetch

### PR DESCRIPTION
**Pull request by qti3e**
This address #596

### CODE REVIEW

File: js/fetch_test.ts [hunk 1]
```diff
@@ -1,9 +1,20 @@
...
   const response = await fetch("http://localhost:4545/package.json");
   const json = await response.json();
   assertEqual(json.name, "deno");
 });
+
+test(async function fetchJsonNetPerm() {
+  let err;
+  try {
+    fetch("http://localhost:4545/package.json");
+  } catch (err_) {
```
Comment (qti3e) [inline js/fetch_test.ts]: @ry Any idea why this catch statement does not work?

Comment (MarkTiedemann) [reply js/fetch_test.ts]: @qti3e The `await` in front of the `fetch(...)` call is missing.

Comment (qti3e) [reply js/fetch_test.ts]: @MarkTiedemann I tried that before...

**Comment by qti3e** [comment]
@ry PTAL

**Pull request merged**